### PR TITLE
create microshift image

### PR DESCRIPTION
**Pull request by resoluteCoder**
- added role to create microshift image
- added ini to gitignore
- created playbook for image role

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: roles/image_builder/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Role Name
```
Comment (maxamillion) [inline roles/image_builder/README.md:1]: none of this content looks relevant to the role does, probably could use an update .... check out the role READMEs we did for the infra.osbuild collection for inspiration

**Review by maxamillion — CHANGES_REQUESTED** [review]
Please update the README, I left an in-line comment about it.

File: roles/image_builder/README.md (hunk 1 above)
Comment (resoluteCoder) [reply roles/image_builder/README.md:1]: Sure thing, it seems like this is from the auto gen from the role command.

I'm on it

File: roles/image_builder/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+# tasks file for image_builder
+- name: Setup osbuild server
+  ansible.builtin.import_role:
+    name: infra.osbuild.setup_server
+
+- name: Running osbuild image builder
+  ansible.builtin.import_role:
+    name: infra.osbuild.builder
```
Comment (maxamillion) [inline roles/image_builder/tasks/main.yml:9]:
```yaml
- name: Running osbuild image builder
  ansible.builtin.import_role:
    name: infra.osbuild.builder
  vars:
    builder_.....: "{{ microshift_image_......  }}"
```

**Review by maxamillion — APPROVED** [review]

File: roles/image_builder/README.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+Requirements
+------------
+
+None
+
+Role Variables
+--------------
+
+### builder_blueprint_name
```
Comment (maxamillion) [inline roles/image_builder/README.md:14]: docs don't appear to match the variable names used below

**Review by maxamillion — CHANGES_REQUESTED** [review]
role readme needs a little fixup

File: roles/image_builder/README.md (hunk 3 above)
Comment (resoluteCoder) [reply roles/image_builder/README.md:14]: One day, one, day. I will complete this PR :laughing: 

Give me one sec :smile:

**Review by maxamillion — APPROVED** [review]

**Pull request merged**